make run-example offline failed

```console
llm-d-kv-cache % make run-example offline
==== Python Configuration ====
Python executable: /opt/homebrew/bin/python3.12 (Python 3.12.12)
Python CFLAGS:     -I/opt/homebrew/opt/python@3.12/Frameworks/Python.framework/Versions/3.12/include/python3.12 -I/opt/homebrew/opt/python@3.12/Frameworks/Python.framework/Versions/3.12/include/python3.12 -fno-strict-overflow -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -O3 -Wall
Python LDFLAGS:    -L/opt/homebrew/opt/python@3.12/Frameworks/Python.framework/Versions/3.12/lib/python3.12/config-3.12-darwin -lpython3.12 -ldl -framework CoreFoundation
==============================
==== Setting up Python virtual environment in /Users/gualiu/github.com/llm-d/llm-d-kv-cache/build/venv ====
Upgrading pip...
Requirement already satisfied: pip in /Users/gualiu/github.com/llm-d/llm-d-kv-cache/build/venv/lib/python3.12/site-packages (26.0.1)
Python virtual environment setup complete.
==== Setting up Python virtual environment in /Users/gualiu/github.com/llm-d/llm-d-kv-cache/build/venv ====
vllm is already installed, skipping...
Checking if ZMQ is already installed...
bash: line 1: pkg-config: command not found
Installing ZMQ dependencies...
✔︎ JSON API formula.jws.json                                                                                                                                                                      Downloaded   31.9MB/ 31.9MB
✔︎ JSON API cask.jws.json                                                                                                                                                                         Downloaded   15.3MB/ 15.3MB
Warning: zeromq 4.3.5_2 is already installed and up-to-date.
To reinstall 4.3.5_2, run:
  brew reinstall zeromq
✅ ZMQ dependencies installed.
Building bin/examples/offline...
github.com/pebbe/zmq4: exec: "pkg-config": executable file not found in $PATH
# github.com/prometheus/client_golang/prometheus
process_collector_mem_cgo_darwin.c:35:33: warning: unused variable 'task' [-Wunused-variable]
make: *** [bin/examples/offline] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -592,16 +592,16 @@ download-zmq: ## Install ZMQ dependencies based on OS/ARCH
 	  echo "Installing ZMQ dependencies..."; \
 	  if [ "$(TARGETOS)" = "linux" ]; then \
 	    if [ -x "$$(command -v apt)" ]; then \
-	      apt update && apt install -y libzmq3-dev; \
+	      apt update && apt install -y libzmq3-dev pkg-config; \
 	    elif [ -x "$$(command -v dnf)" ]; then \
-	      dnf install -y zeromq-devel; \
+	      dnf install -y zeromq-devel pkgconfig; \
 	    else \
 	      echo "Unsupported Linux package manager. Install libzmq manually."; \
 	      exit 1; \
 	    fi; \
 	  elif [ "$(TARGETOS)" = "darwin" ]; then \
 	    if [ -x "$$(command -v brew)" ]; then \
-	      brew install zeromq; \
+	      brew install zeromq pkg-config; \
 	    else \
 	      echo "Homebrew is not installed and is required to install zeromq. Install it from https://brew.sh/"; \
 	      exit 1; \

